src/Raven.Server/Rachis/Leader.cs

```diff
@@ -529,7 +529,7 @@ private void CheckPromotables()
                 if (ambassador.Value.FollowerMatchIndex != lastIndex)
                     continue;
 
-                TryModifyTopology(ambassador.Key, ambassador.Value.Url, TopologyModification.Voter, out Task task);
+                TryModifyTopology(ambassador.Key, ambassador.Value.Url, TopologyModification.Voter, null, out Task task);
```

Comment:
This is the only case we send null, when the key already exists in the dictionary. The key was added when the node became promotable.